benchmarks/ner_embeddings/scripts/multifewerhashembed.py

```diff
@@ -0,0 +1,186 @@
+from typing import Callable, Dict, Tuple, Optional, Any, Union, cast, TypeVar
+from typing import List
+from thinc.api import chain, concatenate, with_array, Model, list2ragged
+from thinc.api import ragged2list, Maxout
+from thinc.initializers import uniform_init
+from thinc.layers.array_getitem import ints_getitem
+from thinc.types import Floats1d, Floats2d, Ints1d, Ints2d, Literal, Ragged
+from thinc.util import partial
+
+from spacy.tokens import Doc
+from spacy.util import registry
+from spacy.ml.staticvectors import StaticVectors
+from spacy.ml.featureextractor import FeatureExtractor
+
+
+InT = TypeVar("InT", bound=Union[Ints1d, Ints2d])
+OutT = Floats2d
+
+
+# TODO: rename
+def FewerHashEmbed(
+    nO: int,
+    nV: int,
+    *,
+    seed: Optional[int] = None,
+    num_hashes: Literal[1, 2, 3, 4] = 4,
+    column: Optional[int] = None,
+    initializer: Callable = uniform_init,
+    dropout: Optional[float] = None
+) -> Model[InT, OutT]:
+    """
+    An embedding layer that uses the “hashing trick” to map keys to distinct values.
```

Comment:
Should the docstring mention that this is a slightly modified version of `MultiHashEmbed` https://spacy.io/api/architectures#MultiHashEmbed ?

Reply:
Agree! I'll fix the quotation marks too